app/services/encrypted_key_maker.rb

```diff
@@ -1,6 +1,10 @@
 class EncryptedKeyMaker
   include Pii::Encodable
 
+  KEY_TYPE = {
+    KMS: 'KMSx',
```

Comment:
Since there's only one value in `KEY_TYPE` for now, should we just make this `KMS_KEY_TYPE = 'KMSx'`? Or is this part of the plan to refactor `EncryptedKeyMaker` to allow locally-encrypted keys too?